wiki/Tournaments/VTSS/1/en.md

```diff
@@ -0,0 +1,1150 @@
+| TowaMarbleSoda ::{ flag=US }:: | 2 | **4** | ::{ flag=UA }:: **Plumbirb** | [#1](https://osu.ppy.sh/community/matches/92208801) |
+| **DeSconTent** ::{ flag=RU }:: | **4** | 0 | ::{ flag=MX }:: B3AT | [#1](https://osu.ppy.sh/community/matches/92208806) |
+| **skit\_3980** ::{ flag=JP }:: | **0** | -1 | ::{ flag=FR }:: Drien95 | *win by default* |
+| **BaIIsackX** ::{ flag=DE }:: | **4** | 0 | ::{ flag=SE }:: Protus | [#1](https://osu.ppy.sh/community/matches/92209401) |
+| hehe\_xddd ::{ flag=US }:: | 3 | **4** | ::{ flag=RU }:: **Dem4eg-** | [#1](https://osu.ppy.sh/community/matches/92211955) |
+| **MrNoLife** ::{ flag=US }:: | **4** | 0 | ::{ flag=BR }:: Wesley\_Voltz | [#1](https://osu.ppy.sh/community/matches/92215067) |
+| **Finny666** ::{ flag=NL }:: | **4** | 1 | ::{ flag=US }:: KirbyCat\_ | [#1](https://osu.ppy.sh/community/matches/92215504) |
+| **HeatherCat** ::{ flag=GB }:: | **4** | 0 | ::{ flag=US }:: MinuteMan | [#1](https://osu.ppy.sh/community/matches/92215510) |
+| **Carprai** ::{ flag=US }:: | **4** | 2 | ::{ flag=PE }:: dybbuk | [#1](https://osu.ppy.sh/community/matches/92218517) |
+| M3ts1s ::{ flag=US }:: | -1 | **0** | ::{ flag=CO }:: **Daluca** | *win by default* |
+| **Sylliant** ::{ flag=US }:: | **4** | 1 | ::{ flag=EC }:: Notmyname2 | [#1](https://osu.ppy.sh/community/matches/92222959) |
+
+Monday, 11 October 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **CristalNova** ::{ flag=MX }:: | **4** | 1 | ::{ flag=SG }:: Ereshukigaru | [#1](https://osu.ppy.sh/community/matches/92236164) |
+| AN0M4LY ::{ flag=US }:: | -1 | **0** | ::{ flag=CL }:: **shigetsu** | *win by default* |
+| **tgame** ::{ flag=JP }:: | **4** | 1 | ::{ flag=JP }:: yurane0 | [#1](https://osu.ppy.sh/community/matches/92245065) |
+| **Utlending** ::{ flag=NO }:: | **4** | 0 | ::{ flag=DE }:: Luckyyfx | [#1](https://osu.ppy.sh/community/matches/92255768) |
+| **WLgummybear** ::{ flag=CA }:: | **4** | 2 | ::{ flag=IE }:: Alpha Ape Man | [#1](https://osu.ppy.sh/community/matches/92264707) |
+| **Leo99Oliver** ::{ flag=BR }:: | **4** | 2 | ::{ flag=MX }:: qwerty346 | [#1](https://osu.ppy.sh/community/matches/92264348) |
+
+### Qualifiers
+
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/10ed1o45tlHa6jMCwy4cqOEdL7dhk211tliKjB7WoI2c/edit) alongside the final standings on which player made it through to the next stage.
+
+## Ruleset
+
+### Overview
+
+1. Vin-Tana Solo Showdown is an osu! 1v1 double-elimination tournament.
+2. All matches will be played with **Score V2** and **NoFail** mod on.
+3. All matches will be played on `Stable` client or any later version that supports Score V2.
+
+### Registration
+
+1. Registered before September 19th, 2021.
+2. Be in between #50,000 and #200,000 in osu! ranking in Bancho (official osu! server) by the time registration ends.
```

Comment:
this sounds odd for two reasons:
- needs the same edit as above
- explicitly points to the official rankings although it's the only kind of tournament that makes its way to the wiki (and osu! forums, too, I hope)

```suggestion
2. All participants must be ranked between #50,000 and #200,000 in osu! by the time registration ends.
```